osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/utils/DlmsHelper.java

```diff
@@ -178,7 +177,7 @@ public DataObject getClockDefinition() {
     }
 
     public DataObject getAccessSelectionTimeRangeParameter(final DateTime from, final DateTime to,
-                                                           final DataObject selectedValues) {
+            final DataObject selectedValues) {
```

Comment:
Je gebruikt hier meervoud voor een enkelvoudig type. Ik vermoed dat je dit doet omdat ``DataObject`` een verzamel object is.

Ik zou dan zoiets doen:
```suggestion
            final DataObject selectedValuesObject) {
```

Reply:
allemaal bestaande code, die nu goed geformatteerd is. Hierdoor lijkt het een change.